### PR TITLE
Updating the codelabs index page

### PR DESCRIPTION
**Pull request by sfshaza2**
Staged: https://sz-flutter-2.firebaseapp.com/docs/codelabs

### CODE REVIEW

File: src/docs/codelabs/index.md [hunk 1]
```diff
@@ -3,109 +3,147 @@ title: Codelabs
...
-#### [Building Beautiful UIs with Flutter]({{site.codelabs}}/codelabs/flutter)
+#### [Write your first Flutter app on the web][]
+
+Implement a simple web app in DartPad (no downloads
+required!) that displays a sign-in screen
+containing three text fields. As the user fills out the
+fields, a progress bar animates along the top of the
+sign-in area. This codelab is written specifically for
+the web, but if you have downloaded and configured
+Android and iOS tooling, the completed app just
```
Comment (kwalrath) [inline src/docs/codelabs/index.md]: The end of the sentence is a little confusing — _just_ could be read as _only_, e.g. How about:

...the completed app works on Android and iOS devices, as well.

File: src/docs/codelabs/index.md [hunk 2]
```diff
@@ -3,109 +3,147 @@ title: Codelabs
...
+works on all of these devices.
 
-A deeper "first dive" than "Write Your First Flutter App." In this codelab
-you'll create a chat app that includes a simple animation, and customizes
-the UI for iOS and Android.
+
+#### [Basic Flutter layout concepts][]
+
+Use DartPad in a browser (no download required!)
```
Comment (kwalrath) [inline src/docs/codelabs/index.md]: Global: no download required -> no downloads required
or vice versa
(Sometimes the page says "no download required", and sometimes it says "...downloads...")

File: src/docs/codelabs/index.md [hunk 3]
```diff
@@ -3,109 +3,147 @@ title: Codelabs
...
+by adding a backdrop with a menu that filters
 products by the selected category.
 
 ---
 
-For a full list of available Flutter codelabs, see the
-[Flutter category]({{site.codelabs}}/?cat=Flutter)
-on [Google Developers]({{site.codelabs}}).
-If the previous link doesn't work, try [this
-mirror of the Flutter codelabs](https://codelabs.flutter-io.cn/).
```
Comment (kwalrath) [inline src/docs/codelabs/index.md]: Should we link to .cn somewhere on this page?

**Comment by sfshaza2** [comment]
Thanks for your review, @kwalrath!

**Pull request merged**